component.yaml

```diff
@@ -0,0 +1,37 @@
+platform: Common
+buildPlatform: ReactNative
+displayName: "React Native Unit Test"
+description: "Run React Native unit tests with Jest"
+inputs:
+  - key: "AC_REPOSITORY_DIR"
+    defaultValue: "$AC_REPOSITORY_DIR"
+    isRequired: true
+    title: Repository Directory
+    description: Specifies the cloned repository directory.
+    helpText:
+  - key: "AC_OUTPUT_DIR"
+    defaultValue: "$AC_OUTPUT_DIR"
+    isRequired: true
+    title: Output Direction
+    description: Specify the output directory.
+    helpText:
+  - key: "AC_RN_TEST_COMMAND_ARGS"
+    isRequired: false
```

Comment:
```suggestion
  - key: "AC_RN_TEST_COMMAND_ARGS"
    defaultValue: "$AC_RN_TEST_COMMAND_ARGS"
    isRequired: false
```

Reply:
@csonuryilmaz said that there is no problem, ıt can be empty.

Reply:
It can be empty by default. No need to fill in with environment variable in this case.